serverless/serverless-kafka.adoc

```diff
@@ -12,6 +12,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 You can use the `KafkaChannel` channel type and `KafkaSource` event source with {ServerlessProductName}.
 To do this, you must install the Knative Kafka components, and configure the integration between {ServerlessProductName} and a supported link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/amq_streams_on_openshift_overview/index[Red Hat AMQ Streams] cluster.
 
+[NOTE]
+====
+Apache Kafka is currently unsupported on IBM Z and IBM Power Systems.
```

Comment:
Can you please add attributes for the IBM product names to make sure they are consistent going forward. Also, same not about "is not supported for" as opposed to "is unsupported"

Reply:
As of now we do not have attributes for IBM products.

Reply:
Please add them, you can reach out to @vikram-redhat or the OCP docs team for guidance on adding attributes.